### DISCUSSION TITLE
IPBan_EmergingThreats_0 and IPBan_EmergingThreats_1000 not created

### DISCUSSION THREAD

**Discussion by waldemarvulf** [DigitalRuby/IPBan#217] [Q&A] [answered]
Hello. IPBan_Block_0 was created when the attack was attempted, but IPBan_EmergingThreats_0 and IPBan_EmergingThreats_1000 were not created when installing ipban on windows server 2019. Although on another computer with windows 10 pro these rules were created.

**Comment by jjxtra** [comment, marked as answer]
In newer versions I commented out the emerging threats rule. You can uncomment it in you firewall uri rules setting.

**Comment by waldemarvulf** [comment]
I did not find commented lines related to firewall rules in the config
file.
 Is it this line?
Value must be true?
<add key="FirewallUriRules" value=""/>



сб, 10 сент. 2022 г., 15:50 Jeff Johnson ***@***.***>:

**Reply by jjxtra** [reply]
You can put this value in to enable emerging threats (after the -->):
```
EmergingThreats,01:00:00:00,https://rules.emergingthreats.net/fwrules/emerging-Block-IPs.txt,10000
```

**Comment by waldemarvulf** [comment]
Thank you Jeff! It works.


сб, 10 сент. 2022 г. в 18:32, Jeff Johnson ***@***.***>:

**Comment by WillyMartin** [comment]
<img width="945" height="224" alt="image" src="https://github.com/user-attachments/assets/592941fd-4201-4627-97d7-15e9e4fc64ab" />

**Reply by jjxtra** [reply]
That XML is malformed. Value should be the rules.

**Reply by WillyMartin** [reply]
> That XML is malformed. Value should be the rules.

thx

2026-02-13 18:41:39.9605|INFO|IPBan|Config file changed
2026-02-13 18:41:40.8375|WARN|IPBan|Updating firewall uri rule EmergingThreats with 1300 ips
2026-02-13 18:41:40.8375|INFO|IPBan|Processing 1 firewall tasks
